src/Groups/action.jl

```diff
@@ -392,6 +386,48 @@ end
 ^(I::MPolyIdeal, p::MatrixGroupElem) = on_indeterminates(I, p)
 
 
+# We do not support `on_lines(line::Vector, x::GAPGroupElem)`
+# because for example `*(::fpFieldElem, ::Vector{fpFieldElem})`
+# is not supported.
+"""
+    on_lines(line::GAP.GapObj, x::GAPGroupElem)
+    on_lines(line::AbstractAlgebra.Generic.FreeModuleElem, x::GAPGroupElem)
+
+Return the image of the nonzero vector `line` under `x`,
+where the action is given by first computing `line * x`
+and then normalizing the result by scalar multiplication from the left
+such that the first nonzero entry is the `one` of the `base_ring` of `line`.
+
+# Examples
+```jldoctest
+julia> n = 2;  F = GF(5);  g = general_linear_group(n, F);
+
+julia> v = gen(free_module(F, n), 1);
+(1, 0)
+
+julia> m = gen(g,2)
+[4   1]
+[4   0]
+
+julia> v * m
+(4, 1)
+
+julia> on_lines(v, m)
+(1, 4)
+```
+"""
+on_lines(line::GapObj, x::GAPGroupElem) = GAPWrap.OnLines(line, x.X)
+
+function on_lines(line::AbstractAlgebra.Generic.FreeModuleElem, x::GAPGroupElem)
+    res = line * x
+    i = 1
+    while iszero(res[i])
+      i = i+1
+    end
```

Comment:
this will error if `line` is zero (probably should not happen and will not happen when used properly)